app/views/questionnaire.py

```diff
@@ -240,17 +240,16 @@ def post_everyone_at_address_confirmation(eq_id, form_type, collection_id, group
     return post_block(eq_id, form_type, collection_id, group_id, group_instance, 'permanent-or-family-home')
 
 
-def validate_all_answers(answer_store, metadata):
+def is_survey_completed(answer_store, metadata):
     path_finder = PathFinder(g.schema_json, answer_store, metadata)
-    error_messages = SchemaHelper.get_messages(g.schema_json)
+    completed_blocks = get_completed_blocks(current_user)
 
     for location in path_finder.get_routing_path():
-        block_json = _render_schema(location)
-        form, _ = get_form_for_location(block_json, location, answer_store, error_messages)
+        if location.block_id in ['thank-you', 'summary', 'confirmation']:
```

Comment:
Will this still work since we moved Intro and Summary etc into Blocks and gave them specific types. Suggest this should be checking the block type not id now?